git_machete/client.py

```diff
@@ -1920,7 +1925,7 @@ def checkout_github_prs(self,
 
         debug('Current GitHub user is ' + (current_user or '<none>'))
         self.__sync_annotations_to_definition_file(all_open_prs, current_user, verbose=verbose)
-        if pr and len(valid_prs) == 1:
+        if pr and len(checked_out_prs) == 1:
```

Comment:
Hmm maybe:
```suggestion
        if pr and len(valid_prs) == 1 and len(checked_out_prs) == 1:
```
(https://github.com/VirtusLab/git-machete/pull/435#discussion_r823567752)

Although this is all TBD, I don't press here